build.sbt

```diff
@@ -15,18 +15,23 @@ lazy val newtype = crossProject(JSPlatform, JVMPlatform).in(file("."))
 lazy val newtypeJVM = newtype.jvm
 lazy val newtypeJS = newtype.js
 
-lazy val catsTests = crossProject(JSPlatform, JVMPlatform).in(file("cats-tests"))
```

Comment:
Unfortunately, I'm not aware of any way to disable JS only for one Scala version (2.11 doesn't have any scala.js 1.0.0 artifact). One could do some black set magic to test Scala 2.11 with scala.js 1.0.0, but as these are only tests and all Scala versions are covered on the JVM, I would rather keep it simple like this.

Reply:
Sounds good for now